### PR TITLE
5-0-7 version history

### PR DESCRIPTION
**Pull request by hflynn**
Based on release announcement

### CODE REVIEW

File: history.txt [hunk 1]
```diff
@@ -1,6 +1,18 @@
 OMERO version history
 =====================
 
+5.0.7 (February 2015)
+---------------------
+
+This is bug-fix release covering a number of issues:
```
Comment (mtbc) [inline history.txt]: +"a"

File: history.txt [hunk 2]
```diff
@@ -1,6 +1,18 @@
 OMERO version history
 =====================
 
+5.0.7 (February 2015)
+---------------------
+
+This is a bug-fix release covering a number of issues:
+
+-  rendering improvements including 32 bit and float support
```
Comment (rleigh-dundee) [inline history.txt]: Hyphenate 32-bit?

**Comment by rleigh-dundee** [comment]
What's there looks fine other than the minor change noted.  However, don't we have more detail we can include here?

**Comment by sbesson** [comment]
--rebased-to #3439

**Pull request merged**